examples/generic_connector_configs/elasticsearch_secretless.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+services:
+  token-service:
+    connector: generic_http
+    listenOn: tcp://0.0.0.0:9030
+    credentials:
+      token:
+        from: keychain
+        get: summon#elasticsearch/access_token
+    config:
+      headers:
+        Authorization: Bearer {{ .token }}
+      forceSSL: true
+      authenticateURLsMatching:
+        - ^http[s]*\:\/\/
+  api-key-service:
+    connector: generic_http
+    listenOn: tcp://0.0.0.0:9020
+    credentials:
+      id:
+        from: keychain
+        get: summon#elasticsearch/api_key_id
```

Comment:
Same here. I wonder if something like `path/to/elasticsearch/api/key/id` and `path/to/elasticsearch/api/key` would be more informative for the example.